src/app/components/core/control-dispatcher/list-control-dispatcher/list-control-dispatcher.component.html

```diff
@@ -1,4 +1,54 @@
 <div class="container is-fluid">
+  <ng-container>
+    <ng-container *ngIf="!dispatcherServiceInstalled">
+      <article class="message is-warning">
+        <div class="message-body">
+          <span>Dispatcher service is not installed.</span>
+          <button (click)="getInstalledServicesList()" class="button is-small" id="refresh-check">
+            <span class="icon">
+              <i class="fa fa-sm fa-sync" aria-hidden="true"></i>
+            </span>
+          </button>
+          <span class="icon is-small tooltip is-pulled-right is-hovered btn-pad" data-tooltip="Help"
+            (click)="goToLink('control-dispatcher-service')">
+            <i class="far fa-question-circle"></i>
+          </span>
```

Comment:
We probably can keep it as single occurrence instead of repeating in all 3 conditions.